src/core.js

```diff
@@ -65,6 +65,12 @@ exports.register([
     require('./components/rangeselector')
 ]);
 
+// locales en and en-US are required for default behavior
+exports.register([
+    require('./locale-en'),
+    require('./locale-en-US')
+]);
```

Comment:
🎩  `bundle_tests/core_test` 🌟 

Since we're constructing our own `d3.locale` now no matter what, `locale-en` needs to be part of the core so it can be used as the fallback for missing pieces of other locales. Then for simplicity I moved the default `locale-en-US` to the core as well, which has the nice side-effect of keeping these redundant locales out of `dist/`.

@etpinard seem reasonable? Are you OK with putting these two in the root `src/` directory? At least as long as we're not going to put any other locales into the core it doesn't seem like they warrant their own directory.

Minor downside is the `lib/index*` files don't demonstrate how to register locales, but we should be able to to better than documenting it there anyhow - somewhere we will want examples of how to use locales both in the build process and via script tags.

Reply:
> @etpinard seem reasonable?

This is very reasonable :ok_hand: 

> but we should be able to to better than documenting it there anyhow 

Yep, we should at least open up an issue on https://github.com/plotly/documentation/issues and / or add a _Locale_ section to `dist/README.md` via [`tasks/stats.js`](https://github.com/plotly/plotly.js/blob/master/tasks/stats.js).

Reply:
updated `dist/README.md` -> https://github.com/plotly/plotly.js/pull/2207/commits/331b2daca8bb6398d01d6b0190e9e1bfba9e980c

will open an issue at https://github.com/plotly/documentation once this is merged.